tests/third_party/cupy/sorting_tests/test_search.py

```diff
@@ -366,7 +366,6 @@ def test_argwhere(self, xp, dtype):
 # return cupy.nonzero(self.array)
 
 
-@testing.gpu
 class TestNanArgMin(unittest.TestCase):
     @testing.for_all_dtypes(no_complex=True)
```

Comment:
for some below tests cupy uses `@testing.for_float_dtypes()`, I think we can align here